WebApp/autoreduce_webapp/templates/help.html

```diff
@@ -68,44 +68,24 @@ <h3>Accessing my reduced data</h3>
-                        <li>Create a new IDAaaS instance for your instrument / technique.</li>
-                        <li>Follow the instructions on the <a href="https://isis.analysis.stfc.ac.uk/#/help">IDAaaS help page</a>
-                            under the <em>Download/Upload Files</em> section that will walk you through the SFTP process </li>
+                        <li>Open an existing or create and open a new IDAaaS virtual machine / instance</li>
+                        <li>The reduced files are then available within the virtual machine within folders named "autoreduced"
+                            within RB folders; and at locations as detailed on completed run pages on the WebApp </li>
                     </ol>
-                    <h4>ISIS Compute</h4>
-                    If you do not currently have your instrument supported by IDAaaS, you should use the ISIS Compute service.
+                    <h4>Use the IDAaaS FileBrowser to copy reduced data elsewhere</h4>
                     <ol>
-                        <li>Open your SFTP client</li>
-                        <li>Host name: <strong>isiscompute.nd.rl.ac.uk</strong></li>
-                        <li>Port: <strong>22</strong></li>
-                        <li>Enter your federal ID and password</li>
-                        <ul>
-                            <li>To avoid repeating these steps in the future, click save and fill in the save dialog box that opens</li>
-                        </ul>
-                        <li>Click Login</li>
-                        <ul>
-                            <li>You may get a warning message about a potential security breach.
-                                This is nothing to worry about and is common, especially if it's your first time accessing the server in a while.
-                                Simply click <strong>Update</strong> to continue.</li>
-                        </ul>
-                        <li>Once the client has connected to the server, you will see your files (default on the left) and the server files (default on the right)</li>
-                        <li>From here you can now copy and paste between your local files and those on CEPH.</li>
-                        <li>You can find the CEPH data at /instrument. See <a href="#reduction_data_location">Where does my reduced data go?</a> for more details.</li>
+                        <li>Visit the page <a href="https://data.analysis.stfc.ac.uk">https://data.analysis.stfc.ac.uk</a> and login</li>
+                        <li>Click on "Data" on the left menu</li>
+                        <li>Reduced data are then available within RB folders you have access to from HOME or by stepping through to
+                            the relevant RB folder (that you have access to) from INSTRUMENT. Within these identify the "autoreduced"
+                            sub-folder, and as detailed on completed run pages on the WebApp</li>
```

Comment:
> and as detailed on completed run pages on the WebApp

Is there a bit of the sentence missing? It might have to be reworded as I'm not sure what it means

Reply:
Thanks for spotting and just submitted a rewording - let me know if more readable